code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -204,6 +204,31 @@
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/engine/atmos
 	tauric = TRUE		//Citadel Add for tauric hardsuits
 
+	//Radiation
+/obj/item/clothing/head/helmet/space/hardsuit/engine/rad
+	name = "radiation hardsuit helmet"
+	desc = "A special helmet that protects against radiation and space. Not much else unfortunately."
+	icon_state = "cespace_helmet"
+	item_state = "nothing"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 0)
+	item_color = "engineering"
+	resistance_flags = FIRE_PROOF
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	actions_types = list()
+
+/obj/item/clothing/suit/space/hardsuit/engine/rad
+	name = "radiation hardsuit"
+	desc = "A special suit that protects against radiation and space. Not much else unfortunately."
+	icon_state = "hardsuit-rad"
+	item_state = "nothing"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 0)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/engine/rad
+	resistance_flags = FIRE_PROOF
+	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	tauric = FALSE		//Citadel Add for tauric hardsuits
+
+/obj/item/clothing/head/helmet/space/hardsuit/engine/rad/attack_self()
```

Comment:
Is this ment to use engi hardsuit icon?

Reply:
It is using its own suit icon, a recolor of old suit, and I didn't want to do a tauric/naga version nor an in-hand version, to the tauric flag is set to false and the icon set to nothing mean it will not show up on the player when holding it in hands.